src/backer/backering.py

```diff
@@ -453,31 +455,29 @@ def on_post(self, req, rep):
         if ilk not in (Ilks.icp, Ilks.rot, Ilks.ixn, Ilks.dip, Ilks.drt):
             raise falcon.HTTPBadRequest(description=f"invalid event type ({ilk})for receipting")
 
-        msg = bytearray(serder.raw)
-        msg.extend(cr.attachments.encode("utf-8"))
+        ### start ###
+        rct = self.hab.receipt(serder)
+        self.psr.parseOne(ims=bytes(rct))
+        wigs = self.hab.db.getWigs(dbing.dgKey(pre, serder.said))
 
-        self.psr.parseOne(ims=msg)
+        # Extend event with wigers
+        event_msg = bytearray(serder.raw)
+
+        event_msg.extend(cr.attachments.encode("utf-8"))
+        event_msg.extend(coring.Counter(code=coring.CtrDex.WitnessIdxSigs,
+                                        count=len(wigs)).qb64b)
+        for wig in wigs:
+            event_msg.extend(wig)
+        ## end ###
+        self.ledger.publishEventv2(event_msg)
+        self.psr.parseOne(ims=event_msg)
```

Comment:
We probably don't need to parse this as long as the original message is parsed like in my other comment and the rct

Reply:
updated
ref here 60b40541ac1ece5dff7ef05ea8725fb0c7342700